pkg/tpkg/registry.go

```diff
@@ -343,7 +343,7 @@ func (gr *gitRegistry) Load(ctx context.Context, sync bool, cache Cache, ui UI)
 			url := gr.url
 
 			var err error
-			for _, branch := range []string{"master", "main", "trunk"} {
+			for _, branch := range []string{"main", "master", "trunk"} {
```

Comment:
Add a comment that explains that the order matters here.

Reply:
added a comment, but since the race-condition will disappear with the follow-up PRs, didn't explicitly say that the order matters (except for speed).